evento/src/views/SignInPage/index.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import Cookie from 'js-cookie';
+import './SignInPage.css'
+
+class SignInPage extends Component {
+	constructor() {
+		super();
+		this.state = { 
+			email: '',
+			password: '',
+		 	errorMessage: ''
+		};
+	}
+
+	handleSubmit(evt) {
+		evt.preventDefault();
+
+		fetch('/authenticate', {
+			method: 'POST',
+			headers: { 'Content-Type': 'application/json', 'Access-Control-Allow-Origin':'*' },
+			body: JSON.stringify({
+				email: this.state.email,
+				password: this.state.password,
+			})
+		}).then(response => {
+			if (response.status === 200) {
+				return response.json();
+			}
+			this.setState({ errorMessage: 'Invalid credentials' });
+		}).then(authKey => {
+			Cookie.set('auth_token', authKey.auth_token);
+
+			// User id needs to be fetched and saved to cookie
+			fetch('/users')
```

Comment:
Sure, but we should implement it in the backend now instead of pushing hacks like that to production